### ISSUE TITLE
[FIX] 10_DownloadData.R Error: object 'Found' not found

### ISSUE THREAD

**Issue by matiarj** [OpenSourceAP/CrossSection#4]
Line 326-329
Reads this:
```
ipos = read_excel(path = tmp) %>% 
  transmute(Founding = Found,
            Offerdate = `Offer date`,
            CRSPperm = PERM)
```

Should Read:
```
ipos = read_excel(path = tmp) %>% 
  transmute(Founding = 'Founding',
            Offerdate = 'Offer date',
            CRSPperm = 'CRSP PERM')
```

**Comment by mk0417** [comment]
Should use backtick (``) to reference variable names with space. Anything in single quotation ('') will be considered as string and it will replace the original values.

This works.

```
ipos = read_excel(path = tmp) %>% 
  transmute(Founding = Founding, 
            Offerdate = `Offer date`,
            CRSPperm = `CRSP perm`)
```

**Comment by OpenSourceAP** [comment]
Thank you both. Fixed in commit 241f9eaf204f558be64cb5f5b84688aac3039273
